concepts/60 Themes and Styles/08 ThemeBuilder/20 Postpone Customization.md

```diff
@@ -1,7 +1,13 @@
-To return to customization later, save the theme's metadata beforehand. Click **Export** on the toolbar to open the **Theme Export** popup dialog. In this dialog, select the **Export Metadata** tab and click **Save to File**. A `.json` file with metadata is saved on your computer. Alternatively, you can copy the metadata from the text area at the bottom:
+To return to customization later, save the theme's metadata beforehand. Click **Export** on the toolbar to open the **Theme Export** popup dialog. In this dialog, enter the color scheme name.
 
-![DevExtreme ThemeBuilder UI: Theme Export popup dialog](/images/Common/ThemeBuilder/themebuilder-themeexport.png)
+![DevExtreme ThemeBuilder UI: Theme Export popup dialog, Enter Name](/images/Common/ThemeBuilder/themebuilder-themeexport-name.png)
 
-To continue customization, click **Import** on the toolbar. In the appeared **Theme Import** dialog, click **Upload File** and select the `.json` file that you exported previously. If you have copied the metadata rather than exported it as a `.json` file, paste it in the text area and click **Upload Metadata**:
+Go to the **Choose Format** step and click the **Download Metadata File** button. 
+
+![DevExtreme ThemeBuilder UI: Theme Export popup dialog, Choose Format](/images/Common/ThemeBuilder/themebuilder-themeexport-format.png)
+
+A `.json` file with metadata is saved on your computer. Alternatively, you can **Copy Metadata**.
+
+To continue customization, click **Import** on the toolbar. In the appeared **Theme Import** dialog, click **Upload File** and select the `.json` file that you exported previously. If you copied the metadata rather than exported it as a `.json` file, paste it in the text area and click **Upload Metadata**:
```

Comment:
```suggestion
To continue customization, click **Import** on the toolbar. In the invoked **Theme Import** dialog, click **Upload File** and select the `.json` file that you exported previously. If you copied the metadata rather than exported it as a `.json` file, paste it in the text area and click **Upload Metadata**:
```